samples/Build2019Demo.Unity/Assets/Generated.StateSynchronization.AssetCaches/Resources/SpriteAssetCache.asset

```diff
@@ -14,176 +14,297 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   assets:
   - AssetId:
-      m_storage: 74fae14d-39c0-497e-ae6d-72f1f2084231
-    Asset: {fileID: 21300000, guid: d651fc00b471cfe44834a222e9433461, type: 3}
+      guid:
+        m_storage: 00000000-0000-0000-f000-000000000000
```

Comment:
>00000000-0000-0000-f000-000000000000 [](start = 19, length = 36)

This GUID seems to be strannnnnnnnnnngely full of zeros - is it just the case that some portion of our deterministic guids are going to look like this?

It's just interesting seeing a bunch of other GUIDs that look like regular GUIDs, and then seeing this guid (and the c000 one) which looks like a bunch of zeroes that then got smashed by a cosmic ray.

Reply:
I have seen guids like this for Unity's default assets. Things like cube and cone meshes have similar guids.